Source/WebCore/svg/SVGAnimationElement.cpp

```diff
@@ -257,7 +258,7 @@ void SVGAnimationElement::beginElement()
 
 void SVGAnimationElement::beginElementAt(float offset)
 {
-    if (std::isnan(offset))
+    if (std::isfinite(offset))
         return;
```

Comment:
I think the check here should be the opposite:

```
  if (!std::isfinite(offset))
        return;
```

Reply:
Given the current IDL, offset should never be infinite (or we would have thrown on bindings side). So while I agree with Said that the check seems backwards, I think we shouldn't even have a check unless this code has other call sites besides the bindings.

Reply:
I opted to remove and it seems to work fine and I have rebaselined all tests now. Thanks for feedback.